lib/karafka.rb

```diff
@@ -32,6 +35,11 @@ def monitor
       @monitor ||= App.config.monitor
     end
 
+    # @return [::Karafka::EventPublisher] event publisher we want to use.
+    def event_publisher
```

Comment:
can we call it just ```events```?  It's recommended by @solnic as well - then we get a nice ```events.publish``` not events_published.publish - sounds a bit railsy :D

Reply:
Good idea!

I'm also wondering whether migrating to dry-events isn't an overkill in case of these callbacks. These are only 2 events in the whole lifecycle of the application.

Are there any other use-cases beyond setting-up 3rd party libraries here?